<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechargeServiceClass</class>
 <widget class="QDialog" name="RechargeServiceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/function_background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_RechargeTitle">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 112, 0);</string>
   </property>
   <property name="text">
    <string>会员充值服务</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RechargeStart">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>执行充值操作</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RechargeShow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>1021</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>充值信息提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_OperationInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>温馨提示：&quot;上翻&quot;和&quot;下翻&quot;键移动光标，&quot;确定&quot;键选中操作</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>301</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/recharge_orderNo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/recharge_orderNo.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_RechargeOrderNO">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_MemberRechargeProcShow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>711</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/member_recharge_p0.png</pixmap>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_RechargeOrderNO</tabstop>
  <tabstop>pushButton_RechargeStart</tabstop>
 </tabstops>
 <resources>
  <include location="../bicycle.qrc"/>
 </resources>
 <connections/>
</ui>
